index.d.ts

```diff
@@ -50,7 +59,7 @@ openEditor([
 ]);
 ```
 */
-export default function openEditor(files: readonly PathLike[], options?: Options): void;
+export default function openEditor(files: readonly PathLike[], options?: Options): void | Promise<void>;
```

Comment:
Returning a conditional Promise is weird. Let's just make the JS code also always be a promise.

Reply:
So what is your suggesstion? To just always return a Promise, even without `wait=true`? So something like 

    return Promise.resolve()

Reply:
No, make the exported function an async function.